src/wallet.ts

```diff
@@ -310,16 +310,12 @@ export function createWalletMiddleware({
     req: JsonRpcRequest<unknown>,
     res: PendingJsonRpcResponse<unknown>,
   ): Promise<void> {
-    const message: string = (req.params as string)[0];
-    const signature: string = (req.params as string)[1];
-    const extraParams: Record<string, unknown> =
```

Comment:
I think there is a reason why we are accepting extra params, and I suspect we may need to keep this for backward compatibility. I would need to confirm on this.

Reply:
Gotcha, I removed it because @metamask/eth-sig-util only takes the [two parameters](https://github.com/MetaMask/eth-sig-util/blob/main/src/personal-sign.ts#L48), but it's possible that the API changed when the package got renamed.

Reply:
It looks like those extra parameters were never used in practice. I looked back through the history all the way to v1.x.

Reply:
Gotcha. I also checked too and this makes sense.